Source: Ingmar Vanhassel
Upstream: Submitted. Bug #4542
Reason: Fix automagic Speex dependency

From f31a47bbc2afc210b55ca1a6f282839f7e22086a Mon Sep 17 00:00:00 2001
From: Ingmar Vanhassel <ingmar@exherbo.org>
Date: Tue, 26 May 2009 19:05:06 +0200
Subject: [PATCH] Allow opt-out from speex dependency

---
 configure.in |    8 +++++---
 1 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/configure.in b/configure.in
index 36740e9..36d0d2a 100644
--- a/configure.in
+++ b/configure.in
@@ -106,14 +106,16 @@ AC_SUBST(AVCODEC_CFLAGS)
 AC_SUBST(AVCODEC_LIBS)
 AC_SUBST(AVCODEC_HEADER)
 
-PKG_CHECK_MODULES(speexdsp, [speexdsp >= 1.2], [HAVE_SPEEXDSP="yes"], [HAVE_SPEEXDSP=""])
-AM_CONDITIONAL(HAVE_SPEEXDSP, test "$HAVE_SPEEXDSP" = "yes")
-
 AC_ARG_WITH([speex],
 	AS_HELP_STRING([--with-speex={builtin|lib|no}],
 		[build speex resampler (built-in code, link with external lib, or no build)]),
 	[PPH=$withval], [PPH="lib"])
 
+if test "x$with_speex" != "xno"; then
+	PKG_CHECK_MODULES(speexdsp, [speexdsp >= 1.2], [HAVE_SPEEXDSP="yes"], [HAVE_SPEEXDSP=""])
+fi
+AM_CONDITIONAL(HAVE_SPEEXDSP, test "$HAVE_SPEEXDSP" = "yes")
+
 USE_LIBSPEEX=""
 if test "$PPH" = "lib"; then
 	if test "$HAVE_SPEEXDSP" = "yes"; then
-- 
1.6.3.1

